psc/Main.hs

```diff
@@ -52,12 +52,12 @@ data InputOptions = InputOptions
 
 compile :: PSCMakeOptions -> IO ()
 compile (PSCMakeOptions inputGlob inputForeignGlob outputDir opts usePrefix) = do
-  input <- concat <$> mapM glob inputGlob
+  input <- globWarningOnMisses "input files" inputGlob
   when (null input) $ do
     hPutStrLn stderr "psc: No input files."
     exitFailure
   moduleFiles <- readInput (InputOptions input)
-  inputForeign <- concat <$> mapM glob inputForeignGlob
+  inputForeign <- globWarningOnMisses "foreign imports" inputForeignGlob
```

Comment:
Would it be alright if I resolved this first then worked on #1268 in a separate PR?